types/json2csv/index.d.ts

```diff
@@ -1,18 +1,32 @@
-// Type definitions for json2csv 4.2
+// Type definitions for json2csv 4.4
 // Project: https://github.com/zemirco/json2csv
-// Definitions by: Juanjo Diaz <https://github.com/juanjoDiaz>
+// Definitions by: Daniel Gooß <https://github.com/dangoo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
+import { TransformOptions } from 'stream';
+
 import { json2csv } from './JSON2CSVBase';
 import JSON2CSVParser from './JSON2CSVParser';
+import JSON2CSVAsyncParser from './JSON2CSVAsyncParser';
 import JSON2CSVTransform from './JSON2CSVTransform';
 
+export default json2csv;
+
 export {
-	json2csv,
-	JSON2CSVParser as Parser,
+    JSON2CSVParser as Parser,
+    JSON2CSVAsyncParser as AsyncParser,
 	JSON2CSVTransform as Transform
 };
 
 // Convenience method to keep the API similar to version 3.X
-export function parse<T>(data: Readonly<T> | ReadonlyArray<T>, opts?: json2csv.Options<T>): string;
+export function parse<T>(
+    data: Readonly<T> | ReadonlyArray<T>,
```

Comment:
`data` can also be a `stream.Readable`.

Reply:
@juanjoDiaz  Just for me to catch up, how does the synchronous parser handle a stream input?

Reply:
My bad. Only the `asyncParser` can. 🙂